workspaces/global-header/plugins/global-header/src/components/HeaderDropdownComponent/ProfileDropdown.tsx

```diff
@@ -46,6 +47,16 @@ export const ProfileDropdown = ({
   const identityApi = useApi(identityApiRef);
   const [user, setUser] = useState<ProfileInfo>();
   const profileDropdownMountPoints = useProfileDropdownMountPoints();
+  const [bgColor, setBgColor] = useState('#3C3F42');
+
+  useEffect(() => {
+    const container = document.getElementById('global-header');
```

Comment:
I like these "pick up the right color" details. But getElementById doesn't feel like the react way to pick up an element, or?

Maybe we you can create a bug that we get rid of this and use a ref prop instead? But it's fine for now.